README.md

```diff
@@ -1,71 +1,152 @@
-- :warning: : module version: `v1.4.0` along with Kubernetes Version: `1.21.x`. It works as expected.
-Marked as a warning because it is not officially supported by [SIGHUP](https://sighup.io).
-- :warning: : module version: `v1.5.0` along with Kubernetes Version: `1.22.x`. It works as expected.
-Marked as a warning because it is not officially supported by [SIGHUP](https://sighup.io).
-- :warning: : module version: `v1.6.0` along with Kubernetes Version: `1.23.x`. It works as expected.
-Marked as a warning because it is not officially supported by [SIGHUP](https://sighup.io).
-- :x: : module version: `v1.1.0` and `v1.2.0` do not work on Kubernetes > `1.19.x` due to this [issue](https://github.com/open-policy-agent/gatekeeper/issues/820).
+| Kubernetes Version |   Compatibility    |                        Notes                        |
+| ------------------ | :----------------: | --------------------------------------------------- |
+| `1.20.x`           | :white_check_mark: | No known issues                                     |
+| `1.21.x`           | :white_check_mark: | No known issues                                     |
+| `1.22.x`           | :white_check_mark: | No known issues                                     |
+| `1.23.x`           |     :warning:      | Conformance tests passed. Not officially supported. |
+
+Check the [compatibility matrix][compatibility-matrix] for additional informations about previous releases of the modules.
+
+## Usage
+
+### Prerequisites
+
+|            Tool             |  Version  |                                                                          Description                                                                           |
+| --------------------------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [furyctl][furyctl-repo]     | `>=0.6.0` | The recommended tool to download and manage KFD modules and their packages. To learn more about `furyctl` read the [official documentation][furyctl-repo]. |
+| [kustomize][kustomize-repo] | `>=3.5.0` | Packages are customized using `kustomize`. To learn how to create your customization layer with `kustomize`, please refer to the [repository][kustomize-repo]. |
+| [KFD Monitoring Module][kfd-monitoring] | `>v1.10.0` | Expose metrics to Prometheus *(optional)* |
+
+> You can comment out the service monitor in the [kustomization.yaml][core-kustomization] file if you don't want to install the monitoring module.
+
+
+### Deployment
+
+1. List the packages you want to deploy and their version in a `Furyfile.yml`
+
+```yaml
+bases:
+  - name: opa/gatekeeper
+    version: "v1.6.0"
+```
+
+> See `furyctl` [documentation][furyctl-repo] for additional details about `Furyfile.yml` format.
+
+1. Execute `furyctl vendor -H` to download the packages
+
```

Comment:
The code snippet in the middle makes the ordered list counter restart from `1`. 
I think it's best to explicitly specify the index `1.`, `2.`, etc...

Reply:
my vscode linter didn't like the 1. 2. ... but as you pointed out it is not working. fixed.